lib/matplotlib/animation.py

```diff
@@ -246,7 +246,7 @@ def frame_size(self):
         width_inches, height_inches = self.fig.get_size_inches()
         return width_inches * self.dpi, height_inches * self.dpi
 
-    def setup(self, fig, outfile, dpi, *args):
+    def setup(self, fig, outfile, dpi, *args, **kwargs):
```

Comment:
Wouldn't removing `*args` be a potential compatibility break? Admittedly, I am having a hard time imagining a scenario where one would have had extra arguments down to this point, but I just wanted to make sure we aren't doing something blindly. It would be nice if I could remember why we had `*args` there in the first place...